### PR TITLE
Clarify process for retrieving a missing issuer public key.

### PR DESCRIPTION
**Pull request by jonathan-r-thorpe**
This fixes the issue raised in #55.

### CODE REVIEW

**Review by lo-simon — APPROVED** [review]
LGTM

File: docs/Validating Access Tokens.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ _(c) AMWA 2021, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
...
-- Verify the Signature, using the issuer's public key, to ensure the token has not been tampered with. In event the issuer [public key](https://specs.amwa.tv/is-10/releases/v1.0.0/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) is not known, fetch the missing public key via the token `iss` claim as specified in [RFC 8414](https://tools.ietf.org/html/rfc8414 "OAuth 2.0 Authorization Server Metadata") section 3, and temporarily respond with an HTTP 503 code in order to avoid blocking the incoming authorized request.
+- Verify the Signature, using the issuer's public key, to ensure the token has not been tampered with.
+- In event the issuer [public key](https://specs.amwa.tv/is-10/releases/v1.0.0/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) is not known, temporarily respond with an HTTP 503 code in order to avoid blocking the incoming authorized request, and then fetch the missing public key from the server metadata's `jwks_uri` endpoint.  The server metadata can be retrieved via the token `iss` claim as specified in [RFC 8414](https://tools.ietf.org/html/rfc8414 "OAuth 2.0 Authorization Server Metadata") section 3.
```
Comment (garethsb) [inline docs/Validating Access Tokens.md]: Is _temporarily_ maybe a confusing word? Does it means just to this request? If there is a second request after the first one but before the missing key is retrieved, I assume that request will get a `503` but needn't initiate a second fetch.

Deleting both _temporarily_ and _then_ would actually help, maybe?

Btw, do we normally style HTTP response status codes like "`503` (Service Unavailable)"?

Comment (jonathan-r-thorpe) [reply docs/Validating Access Tokens.md]: > Btw, do we normally style HTTP response status codes like "503 (Service Unavailable)"?

Yes, you're right that in IS-04 and IS-05 this convention is followed.

However, this isn't carried forward into, for instance, [IS-08](https://github.com/AMWA-TV/is-08/blob/ecf2a56f7b1c02512cb1205b1488755305f3c490/docs/Behaviour.md?plain=1#L255) or [IS-10](https://github.com/AMWA-TV/is-10/blob/a7fb70b0327aab1b1e54f7b4fa4868e1f5209993/docs/Behaviour%20-%20Resource%20Servers.md?plain=1#L26).

But, given IS-04 and IS-05 are the "gold standard" WRT formatting I'll make the change.

**Review by cristian-recoseanu — APPROVED** [review]
Looks good

**Pull request merged**